mifosng-android/build.gradle

```diff
@@ -249,6 +249,8 @@ dependencies {
 
     // androidx annotations
     implementation 'androidx.annotation:annotation:1.1.0'
+
+    implementation 'com.first.avatargenerator:AvatarImageGenerator:1.4'
```

Comment:
Just add a comment on line 252 it will help contributors understand for what purpose this library is used

Reply:
Sorry to ask, but i am new to git, so i have to commit, rebase and forch push again or anything else is possible?

Reply:
1. Add new commit to [fix_#1591](https://github.com/PatelVatsalB21/android-client/tree/fix_%231591).
2. Squash the recent commit, using rebase `git rebase -i HEAD~2`.
3. Force push `git push origin +fix_#1591`